### PR TITLE
Disallow minting 0 coins, fix #1988

### LINKED ISSUE #1988: [Bug] Mint 0 coins to account
# 🐛 Bug
Command should not permit to mint 0 coins. command account mint invalid values for 0 and -0 Terminal output displays Minting coins
## To reproduce
account mint 0 -0
account mint 0 0

## Expected Behavior
Should display [ERROR] Error minting coins: Invalid value

## System information
Ubuntu Linux

~ Tariq

### PR DESCRIPTION
**Pull request by thefallentree**

### CODE REVIEW

File: client/src/client_proxy.rs [hunk 1]
```diff
@@ -294,6 +294,8 @@ impl ClientProxy {
         let receiver = self.get_account_address_from_parameter(space_delim_strings[1])?;
         let num_coins = Self::convert_to_micro_libras(space_delim_strings[2])?;
 
+        ensure(num_coins > 0, "Invalid number of coins to mint.");
```
Comment (sunmilee) [inline client/src/client_proxy.rs]: think it should be `ensure!`

**Comment by thefallentree** [comment]
haha, you got me, yes!

File: client/src/client_proxy.rs (hunk 1 above)
Comment (thefallentree) [reply client/src/client_proxy.rs]: haha, you got me, yes!

**Pull request closed without merging**